controllers/schools.js

```diff
@@ -176,7 +177,7 @@ router.all('/', function (req, res, next) {
             }
         }).then(data => {
 
-            let provider = getStorageProviders();
+            let provider = getStorageProviders(res);
```

Comment:
However, res is not used inside that function

Reply:
Yes it is?